fastly/block_fastly_service_package.go

```diff
@@ -72,6 +91,14 @@ func (h *PackageServiceAttributeHandler) Process(_ context.Context, d *schema.Re
 	return nil
 }
 
+type PkgType int64
+
+const (
+	PkgUndefined PkgType = iota
```

Comment:
This threw me off for a bit, completely optional 
```suggestion
	_ PkgType = iota
```